prow/config/jobs/metal3-io/ironic-standalone-operator.yaml

```diff
@@ -48,3 +48,16 @@ presubmits:
           value: "/"
         image: ghcr.io/yannh/kubeconform:v0.6.7-alpine@sha256:824e0c248809e4b2da2a768b16b107cf17ada88a89ec6aa6050e566ba93ebbc6
         imagePullPolicy: Always
+  # name: metal3-dev-env-integration-test-{image_os}-main
+  - name: metal3-dev-env-integration-test-ubuntu-main
```

Comment:
Shouldn't these jobs set USE_IRSO?

Reply:
Afaiu USE_IRSO should be set from jjb side, haven't seen any jenkins jobs setting its own env var in prow. But I'm not a prow pro :smile: Maybe @lentzi90 could help answering this?

Reply:
Yes that should come from jjb. Prow will only trigger the jobs with whatever defaults they have